samples/matter/template/sample.yaml

```diff
@@ -3,20 +3,21 @@ sample:
   name: Matter Template
 tests:
   sample.matter.template.debug:
+    sysbuild: true
     build_only: true
     extra_args: CONFIG_NCS_SAMPLE_MATTER_OPERATIONAL_KEYS_MIGRATION_TO_ITS=y
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
       - nrf7002dk/nrf5340/cpuapp
-      - nrf54l15pdk/nrf54l15/cpuapp
```

Comment:
@nordicjm can you update the sample.yaml and add nrf54l platform to `sample.matter.template.debug`

Reply:
Done